app/templates/sidebar.html

```diff
@@ -68,6 +68,9 @@
             <a href="/manageBonner" class="nav-link text-white {{ 'active' if request.path =='/manageBonner' }}">
               Bonner Management
             </a>
+            <a href="/spreadsheetMaker" class="nav-link text-white d-none {{ 'active' if request.path =='/spreadsheetMaker' }}">
```

Comment:
I feel like this shouldn't be in here if it is hidden. Can we move this code out into its own PR?